docs/release-notes/november-2023.md

```diff
@@ -0,0 +1,138 @@
+- **Semgrep Editor:** Rules created in the editor are private by default. This means only members of your organization can view rules you have created. To create a private rule visible only to you (an individual), ensure that you create the rule in your **individual account**. <!-- 11267 -->
+- `semgrep scan --config <product>` now uses the same endpoint as the `semgrep ci` to fetch the scan configuration. <!-- todo effect of this change -->
+- Improved error pages.
+
+
+### Fixed
+
+- **API:** Fixed an issue where the severities filter did not return the correct value. <!-- gh-11307 -->
+- **CLI tool:**
+    - The `--severity=[VALUE]` option, which can be added to a `semgrep scan` command, has been fixed. <!-- gh-9062 -->
+    - The `--sarif` flag no longer crashes when Semgrep itself encounters errors.
+- Semgrep now refuses to run incompatible versions of the Pro Engine, rather than crashing and returning a confusing error message. <!-- (gh-8873) -->
+- Fixed an issue where the CI provider icons disappeared from the **Scan new project in CI** window. The icons now appear. <!-- 11228 -->
+- Implemented minor fixes for the new onboarding flow. <!-- 11209, 11207 -->
+
+## 💻 Semgrep Code
+
+### Added
+
+- **Findings details page:** For users that have granted code access to Semgrep, you can now view **in-app code snippets** in Semgrep Cloud Platform.
+
+<!-- todo add a screenshot -->
+
+### Changed
+
+- **Scanning timeout:** The timeout per rule and per file has increased from 2 seconds to 5 seconds.
+
+### Fixed
+
+- **Findings page:** Fixed an issue where filtering by repositories wasn't working. <!-- (11414) -->
+
+## ⛓️ Semgrep Supply Chain
+
+### Fixed
+
+-  **Slack messages:**
+    - Improved readability of Semgrep Supply Chain messages by adding new lines between sections. <!-- (11396) -->
+    -  Fixed links that were not working. <!-- (11210) -->
+- Fixed out-of-bounds list access error in `Cargo.lock` parser. <!-- (sc-1072) -->
+
+## 🔐 Semgrep Secrets (beta)
+
+### Added
+
+- Added an optional `--no-secrets-validation` flag to skip secrets validation. To run a Secrets scan without validation, use the command `semgrep ci --secrets --no-secrets-validation`.
+- **Secrets and Secrets details page:** Added a **<i class="fa-solid fa-ticket"></i> ticket icon** to quickly inform users if a ticket has been created for the finding.
+
+### Changed
+
+- Semgrep Secrets now bypasses targets defined in `.semgrepignore`. This means that files not typically part of a SAST or SCA scan scope, such as configuration files, are now scanned by Semgrep Secrets. Broadening the scope of Semgrep Secrets scans means it is more likely to find leaked secrets.
+    - Previously, Semgrep Secrets excluded targets from `.semgrepignore`. Your findings count may increase with this change.
+    - You can still define exclusions from Secrets scanning. To **exclude targets** from Secrets scanning, define files or paths for exclusion in Semgrep Cloud Platform:
```

Comment:
@akuhlens does the targeting work you did respect exclusions set in the platform?

Reply:
Sounds like we're keeping this in then :)